File: HM/26_RemoveDuplicatesFromSortedArray1.py
def removeDuplicates(nums: [int]) -> int:
    nums[:] = sorted(set(nums))
    return len(nums)

def removeDuplicates2(nums):
    idx = 1
    for i in range(1, len(nums)):
        if nums[i] != nums[i-1]:
            nums[idx] = nums[i]
            idx += 1
    return idx    

File: HM/26_RemoveDuplicatesFromSortedArray1.c
int removeDuplicates(int* nums, int numsSize){
    int idx = 1;
    for (int i=1; i < numsSize; i++){
        if (nums[i] != nums[i-1]){
            nums[idx] = nums[i];
            idx++;
        }
    }
    return idx;
}